applications/tasks/src/kz_task_noinput_worker.erl

```diff
@@ -124,22 +124,29 @@ new_state_after_writing(WrittenSucceeded, WrittenFailed, State) ->
 is_task_successful(TaskId, API, ExtraArgs, IterValue) ->
     case tasks_bindings:apply(API, [ExtraArgs, IterValue]) of
         ['stop'] -> 'stop';
-        [{'ok', _Data}=NewIterValue] ->
-            %% For initialisation steps. Skeeps writing a CSV output row.
+        ['ok'] ->
+            {'true', 0, IterValue};
+        [{'ok', NewIterValue}] ->
             {'true', 0, NewIterValue};
-        [{[_|_]=NewRowOrRows, _Data}=NewIterValue] ->
-            Written = store_return(TaskId, NewRowOrRows),
-            {'true', Written, NewIterValue};
-        [{?NE_BINARY=NewRow, _Data}=NewIterValue] ->
-            Written = store_return(TaskId, NewRow),
-            {'true', Written, NewIterValue};
         [{'EXIT', _}] ->
             kz_util:log_stacktrace(),
             Written = store_return(TaskId, ?WORKER_TASK_FAILED),
             {'false', Written, 'stop'};
-        [{Error, _Data}=NewIterValue] ->
+        [{NewRowOrRows, NewIterValue}] when is_list(NewRowOrRows) ->
+            Written = store_return(TaskId, NewRowOrRows),
+            {'true', Written, NewIterValue};
+        [{NewRow, NewIterValue}] when is_binary(NewRow) ->
```

Comment:
Just try to make kz_task_worker & kz_task_noinput_worker follow the same coding style.
Should i change them both to use ?NE_BINARY?